contracts/test/Box.sol

```diff
@@ -1,33 +1,19 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.0;
 
-/**
-    @title A storage box for a string.
-    @notice The storage box can store a single string value, emit an event and also retrieve the stored value.
-    @dev Event emitted on storing the value.
- */
-contract Box {
-    string private _value;
-
-    event Store(string value);
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
```

Comment:
question: do you need these imports?

As those imports are already included in `BaseBox`, things should all work when you remove them in this `Box` contract, no?

Reply:
Nope they were in fact stale imports. removed.